src/backend/interfaces/REST/py.server/caliopen_api/user/tag.py

```diff
@@ -33,51 +26,65 @@ def __init__(self, request, context):
-        user = CoreUser.get(self.user.user_id)
-        user_tags = user.tags
-        tags = []
-        for tag in user_tags:
-            tags.append({'tag_id': tag.tag_id,
-                         'name': tag.name,
-                         'type': tag.type})
-        return {'tags': tags, 'count': len(tags)}
+        # LEGACY CODE. ROUTE MOVED TO API V2
+        # """Return list of system and user tags."""
+        # user = CoreUser.get(self.user.user_id)
+        # user_tags = user.tags
+        # tags = []
+        # for tag in user_tags:
+        #     tags.append({'tag_id': tag.tag_id,
+        #                  'name': tag.name,
+        #                  'type': tag.type})
+        # return {'tags': tags, 'count': len(tags)}
+        raise HTTPMovedPermanently(location="/V2/tags")
 
     @view(renderer='json', permission='authenticated')
     def get(self):
-        """Get a simgle user tag."""
-        name = self.request.swagger_data["tag_id"]
-        tag = UserTagObject(self.user.user_id, name=name)
-        tag.get_db()
-        tag.unmarshall_db()
-        return tag.marshall_json_dict()
+        # LEGACY CODE. ROUTE MOVED TO API V2
+        # """Get a simgle user tag."""
+        # name = self.request.swagger_data["tag_id"]
+        # tag = UserTagObject(self.user.user_id, name=name)
+        # tag.get_db()
+        # tag.unmarshall_db()
+        # return tag.marshall_json_dict()
+        tag_id = self.request.swagger_data["tag_id"]
+        tags_url = self.request.route_path("Tag", tag_id=tag_id)
+        tags_url = tags_url.replace("/v1", "/v2")
+        raise HTTPMovedPermanently(location=tags_url)
 
     @view(renderer='json', permission='authenticated')
     def delete(self):
-        """Delete an user tag."""
+        # LEGACY CODE. ROUTE MOVED TO API V2
+        # """Delete an user tag."""
+        # tag_id = self.request.swagger_data["tag_id"]
+        # tag = UserTagObject(self.user.user_id, tag_id=tag_id)
+        # try:
+        #     tag.get_db()
+        # except NotFound:
+        #     raise HTTPNotFound()
+        # except Exception as exc:
+        #     log.err('Unexpected exception {} when retrivieng user tag'.
+        #             format(exc))
+        #     raise HTTPInternalServerError(exc)
+        # tag.delete_db()
+        # raise HTTPOk
         tag_id = self.request.swagger_data["tag_id"]
-        tag = UserTagObject(self.user.user_id, tag_id=tag_id)
-        try:
-            tag.get_db()
-        except NotFound:
-            raise HTTPNotFound()
-        except Exception as exc:
-            log.err('Unexpected exception {} when retrivieng user tag'.
-                    format(exc))
-            raise HTTPInternalServerError(exc)
-        tag.delete_db()
-        raise HTTPOk
+        tags_url = self.request.route_path("Tag", tag_id=tag_id)
+        tags_url = tags_url.replace("/v1", "/v2")
+        raise HTTPMovedPermanently(location=tags_url)
```

Comment:
No need of this redirection logic as swagger does not know anymore on /v1/tags related routes.

May remove this python API and don't forget to not configure it in config.y